.travis.yml

```diff
@@ -1,2 +1,2 @@
 language: node_js
-node_js: 4.0
+node_js: 8.0
```

Comment:
Looks like tests are not being run. Try changing this to `node_js: 8` so it matches a newest version, I think you may have introduced some language feature on your changes missing on `8.0`.